runhouse/rns/function.py

```diff
@@ -49,6 +52,7 @@ def __init__(
         self.setup_cmds = setup_cmds or []
         self.access = access or self.DEFAULT_ACCESS
         self.dryrun = dryrun
+        self.resources = resources or self.DEFAULT_RESOURCES
```

Comment:
I'd actually rather leave this as None client-side by default. If we change the default resource behavior (I think we will) any functions which are saved with these resources will break.